channel/src/main/java/com/synopsys/integration/alert/channel/jira/server/JiraServerChannel.java

```diff
@@ -54,7 +53,7 @@ protected IssueTrackerContext getIssueTrackerContext(DistributionEvent event) th
 
     @Override
     public IssueTrackerResponse sendRequests(IssueTrackerContext context, List<IssueTrackerRequest> requests) throws IntegrationException {
-        JiraServerRequestDelegator jiraServerService = new JiraServerRequestDelegator(getGson(), context);
+        JiraServerRequestDelegator jiraServerService = new JiraServerRequestDelegator(new Gson(), context);
```

Comment:
Everything else passes in a gson object but this one creates its own, is that intended?

Reply:
This class will be deleted. I just needed it to compile safely for now.